### PR TITLE
Task_9948: Create UI Home Screen

### PR DESCRIPTION
**Pull request by khanhnn-1791**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9948
### Goals :soccer:

+ Navigation Title
+ Genres/ Popular/ New/ see more label
+ Genres/ Popular/ New using CollectionView
+ Genres/ Popular/ New Item using CollectionViewCell
+ Genres/ Popular/ New item include:
- Rating stars using UIView and UIImageView
- Author/ title Label

### Implementation Details :construction:

Screen Shot 2019-04-09 at 1 20 33 PM

### CODE REVIEW

**Comment by khanhnn-1791** [comment]
ready

File: ios-i-book-lover/Scenes/Cell/RatingView/RatingView.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        buttons.append(star1)
+        buttons.append(star2)
+        buttons.append(star3)
+        buttons.append(star4)
+        buttons.append(star5)
+        let number = Int(floor(rate))
+        for i in 0..<number {
+            buttons[i].isSelected = true
+        }
+        for i in number..<5 {
```
Comment (tienpx3010) [inline ios-i-book-lover/Scenes/Cell/RatingView/RatingView.swift]: Em viết thế này thì trường hợp nửa ngôi sao rating làm thế nào?

**Review by tgdong2296 — APPROVED** [review]

File: ios-i-book-lover/Scenes/Home/HomeViewController.swift [hunk 2]
```diff
@@ -9,10 +9,86 @@
...
+    private var newBooksList = [Book]()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+            prepareUI()
+        }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        if let genreLayout = genreCollectionView.collectionViewLayout as? UICollectionViewFlowLayout {
```
Comment (tienpx3010) [inline ios-i-book-lover/Scenes/Home/HomeViewController.swift]: Cái này set trong storyboard được.

File: ios-i-book-lover/Scenes/Cell/GenreCell/GenreCell.swift [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+import UIKit
+import Then
+
+final class GenreCell: UICollectionViewCell {
+    @IBOutlet weak var genreLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        genreLabel.do {_ in
```
Comment (tienpx3010) [inline ios-i-book-lover/Scenes/Cell/GenreCell/GenreCell.swift]:
```suggestion
        genreLabel.do { _ in
```

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**